src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 7) {
+    return 230;
+  }
+
+  if (days === 6) {
+    return 220;
+  }
+
+  if (days === 3) {
```

Comment:
The condition checks for exactly 3 days, which is not correct. The discount should apply for 3 or more days but less than 7 days. Also, avoid using a hard-coded return value.